apps/rewards/app/utils/constants.js

```diff
@@ -26,3 +26,6 @@ export const DISBURSEMENT_UNITS = [
   YEARS,
 ]
 export const OTHER = 'Other…'
+export const PENDING = Symbol('PENDING')
+export const READY = Symbol('READY')
+export const CLAIMED = Symbol('CLAIMED')
```

Comment:
Although we haven't used `Symbol` before, I think it's the way to go whenever possible.